nautobot/extras/datasources/git.py

```diff
@@ -421,6 +421,8 @@ def import_config_context(context_data, repository_record, job_result, logger):
                         grouping="config contexts",
                         logger=logger,
                     )
+        else:
+            context_record.schema = None
```

Comment:
need to set `modified = True` in this case, if and only if this is actually a change to the data.